common/src/Logging.Common.cs

```diff
@@ -62,10 +62,11 @@ public static class Keywords
         [NonEvent]
         public static void Enter(object thisOrContextObject, FormattableString formattableString = null, [CallerMemberName] string memberName = null)
         {
-            DebugValidateArg(thisOrContextObject);
-            DebugValidateArg(formattableString);
             if (IsEnabled)
             {
+                DebugValidateArg(thisOrContextObject);
```

Comment:
We need to validate args only if logging is enabled.

Reply:
Thank you for fixing all that